src/components/AddItems.js

```diff
@@ -39,7 +40,7 @@ const AddItemsToList = () => {
     event.preventDefault();
 
     if (shoppingListItemName === '') {
-      alert('Please enter item name...');
+      alert('please add an item');
```

Comment:
I like the previous capitalization better ("Please") :slightly_smiling_face:

Reply:
Yes thanks, I agree, I will amend it!